docker-compose.yaml

```diff
@@ -16,7 +16,7 @@ services:
     env_file:
       - .env
     ports:
-      - '3000:3000'
+      - '3001:3001'
```

Comment:
for now Dockerfile exposes port `3000`, not `3001`

Reply:
Sorry, thank you, for reminding, I have a very important process running on the port,  in the next commit I will replace it.